lib/TPP/Conversion/ConvertVectorToXsmm/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+add_mlir_pdll_library (
```

Comment:
Do we really need one PDLL library per pattern? This sounds like a huge scalability problem.

Reply:
I didn't say we need one per pattern, we just need one for transpose and then one for the rest of them to make sure transpose runs before the others do. This is so that we can insert and translate transposes before gemms/brgemms.

Reply:
As discussed, we'll be moving transposes and other vector ops into a single pdll file for now.